components/molecules/CardProfile/card-profile.tsx

```diff
@@ -14,27 +17,29 @@ interface CardProfileProps {
 
 const CardProfile = ({ githubAvatar, githubName, totalPRs, dateOfFirstPR }: CardProfileProps): JSX.Element => {
   return (
-    <a href={`https://www.github.com/${githubName}`} target="_blank" className="flex items-center gap-2" rel="noreferrer">
-      <Avatar size={40} avatarURL={githubAvatar ? githubAvatar : undefined} />
-      <div>
+    <Link href={`/user/${githubName}`} className="flex items-center gap-2">
+      <a>
+        <Avatar size={40} avatarURL={githubAvatar ? githubAvatar : undefined} />
```

Comment:
If there is no avatar, this component should not be shown at all, the anchor wrapping needs minor fixing as well!

Reply:
Ok, note that if there is no avatar, a placeholder will be shown: `avatarURL ? "" : "bg-orange-500"`.
Do you think I need to add a condition here? `{githubAvatar && <Avatar />}`

Reply:
Sorry, but I don't understand what you mean about " the anchor wrapping needs minor fixing as well!"

Reply:
Update: if an error occur, it won't show up
```tsx
  const { data: insight, isError } = useInsight(insightId);
  ...
  if (isError) {
    console.log("Error fetching insight", isError);
    return null;
  }
```